Server/src/main/java/local/ModelDAOImpl.java

```diff
@@ -19,10 +19,10 @@
 import java.util.ArrayList;
 
 public class ModelDAOImpl extends DAO implements ModelDAO {
-    private static final String UPDATE_SERIALIZE_OBJECT = "UPDATE trained_model SET retrain = false, serialized_model  = ? WHERE model_id = ? AND asset_type_id = ?";
+    private static final String UPDATE_SERIALIZE_OBJECT = "UPDATE trained_model SET retrain = false, serialized_model  = ? WHERE model_id = ? AND asset_type_id = ? and status_id = ?";
     private static final String GET_SERIALIZE_OBJECT = "SELECT * FROM trained_model WHERE retrain = true";
-    private static final String GET_MODEL_NAME_FROM_ID = "SELECT name from trained_model, model where trained_model.model_id = model.model_id and asset_type_id = ?";
-    private static final String GET_MODEL_FROM_ASSET_TYPE = "SELECT * FROM trained_model WHERE asset_type_id = ?";
+    private static final String GET_MODEL_NAME_FROM_ID = "SELECT name from trained_model, model where trained_model.model_id = model.model_id and model.model_id = ?";
```

Comment:
Maybe the SELECT should be changed to a SELECT DISTINCT because of the change from asset type id to model ID matching. The reason is that this query might return multiple times the same value since different asset type might be using the same model strategy in the trained model table

Reply:
no, but the trained  model table is no longer necessary, so i'm going to update the query to simply get the name from the model table